x-pack/test/apm_api_integration/basic/tests/feature_controls.ts

```diff
@@ -149,13 +149,6 @@ export default function featureControlsTests({ getService }: FtrProviderContext)
         log.error(JSON.stringify(res, null, 2));
       },
     },
-    {
-      req: {
-        url: `/api/apm/settings/custom_links`,
-      },
-      expectForbidden: expect404,
-      expectResponse: expect200,
-    },
```

Comment:
Why did you remove it from here?

Reply:
because this is the basic test and now it throws an error due to the license. And I removed because this is already been checked here https://github.com/elastic/kibana/pull/77630/files#diff-c803b0a623528379cc664f76067bff58R15

Reply:
All endpoints should still have a basic sanity check here. If it throws that's what we should assert.

Reply:
... although looking at this test I do find it a little confusing...